core/EE_Payment_Processor.core.php

```diff
@@ -208,7 +208,7 @@ public function get_ipn_url_for_payment_method($transaction, $payment_method)
      * @param boolean            $update_txn           whether or not to call
      *                                                 EE_Transaction_Processor::update_transaction_and_registrations_after_checkout_or_payment()
      * @param bool               $separate_IPN_request whether the IPN uses a separate request ( true like PayPal )
-     *                                                 or is processed manually ( false like Mijireh )
+     *                                                 or is processed manually ( false like Mijireh used to )
```

Comment:
I think we might be able to use TransFirst or Aut.Net Accept here for the reference, although those don't go bundled with EE core.

Reply:
yeah good point, although each of those probably doesn't have a super long shelf-life now either...